regtests/bin/matrix_ncep

```diff
@@ -179,7 +212,12 @@
   export  multi06='y' # mww3_test_06 (curvilinear grid tests)
   export  multi07='y' # mww3_test_07 (unstructured grid tests)
   export  multi08='y' # mww3_test_08 (wind and ice tests)
-
+  export      ufs='y' # The Unified Forecast System
+  export     grib='y' # grib file field output
+  export rstrt_b4b='y' # Restart Reproducibility
+  export  npl_b4b='y' # MPI task Reproducibility
+  export  nth_b4b='y' # Thread Reproducibility
+  export     esmf='n' # ESMF coupling
```

Comment:
This is added but turned off? Is this holding a place for future updates?

Reply:
Yes, with esmf='y', we could test the compilation of ww3_multi_esmf, but then I realized we cannot execute it. At first, I thought the regtests/bin/run_esmf_test_suite script that Tim had added was working, so I wanted to use the same functionality. It did not work as expenced. so I left it here for future development. Instead, I started #368 which seems an alternative solution. We can discuss it in a tag up.